source/agora/node/Ledger.d

```diff
@@ -475,6 +477,95 @@ version (unittest)
     }
 }
 
+
+// Generates a genesis block containing the validator's enrollment data.
+version (unittest)
+private const(Block) makeGenesisBlock (immutable(ConsensusParams) params)
```

Comment:
Is this version of `makeGenesisBlock` the same as the one in `Base.d`? If we need to reuse this code, we should move it to a common module like `agora.utils.Test`.

Reply:
The following error occurred and could not be compiled: I think 907 will solve this part.
So we'd better put makeGenesisBlock here and fix it when 907 is far away. What do you think?
Cyclic dependency between module constructors/destructors of agora.consensus.ValidatorSet and agora.consensus.EnrollmentManager

Reply:
Alright.